### ISSUE TITLE
MRBS shows logout when no user is logged, can't login

### ISSUE THREAD

**Issue by Fejrix** [dorianim/modern-mrbs-theme#11]
Hi, having problems with a modern theme seems like everyting works just fine after renaming .inc.php files to just .inc but the header.inc is having a problem. Primary it shows this error:

`E_USER_WARNING in C:\xampp\htdocs\web\lib\MRBS\Form\Form.php at line 120`

`Changing the form method after the form has been created may result in the MAX_FILE_SIZE hidden input coming after a file input, thus making it unusable by the server.
MRBS GET: Array
(
)
MRBS POST: Array
(
)
MRBS SESSION: Array
(
    [csrf_token] => 3ceb5828520f445e064fd53d56e1d42308c6a474b5efc9eaee8ee45272681379
) `

![image](https://github.com/user-attachments/assets/7ad1a87f-b8b3-467e-9ea9-2c413210c388)

when i disable this error i get the logout button but everything else except login works, any idea how to fix this? would be really shame to not use this pretty theme. Thank you

**Comment by Fejrix** [comment]
> Does it work with the default theme?

yes when i switch to default in config.php everything is working flawlessly

**Comment by dorianim** [comment]
Since I don't have a XAMPP setup, I can't really test this. If you find a solution, I'd be happy to accept a pull request, though.
